app/Helper/helper.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/**
+ * Generates random token
+ *
+ * @param $email
+ *
+ * @return string
+ */
+function generateToken($email)
+{
+    return md5(microtime().$email);
+}
+
+/**
+ * Checks if checkbox named all is selected
+ *
+ * @param $name
+ * @param $list
+ *
+ * @return array
+ */
+function isAllSelected($name, $list)
+{
+    if ($name) {
+        return ["ALL"];
+    } else {
+        $list = ($list == "") ? ["ALL"] : $list;
+
+        return $list;
+    }
+}
+
+/**
+ * Checks if token is valid
+ *
+ * @param $email
+ * @param $token
+ *
+ * @return bool
+ */
+function isTokenValid($email, $token)
+{
+    try {
+        $subscriber = new \App\Http\Services\SubscriberService();
+        $res = $subscriber->getSubscriberWithEmailToken($email, $token);
+        return $res;
+    } catch (\Exception $e) {
+        return false;
+    }
+}
+
+/**
+ * Returns subscribed countries
+ *
+ * @param $subscriber
+ *
+ * @return mixed
+ */
+function getSubscribedCountry($subscriber)
+{
```

Comment:
You can define [Accessor](https://laravel.com/docs/5.3/eloquent-mutators#accessors-and-mutators) in subscribe model instead of creating function in helper file.